doc/dev/kdb-operations.md

```diff
@@ -22,33 +24,32 @@ The basic flow of this operation is:
 
 1. Create empty `KDB` instance
 2. Configure `KDB` instance for bootstrap
-3. Run bootstrap `get` operation
-4. Setup default global plugins <!-- TODO: maybe change name; global plugins are very different from normal plugins (although the C API is the same)-->
-5. Process contract
-6. Parse mountpoints
-7. Configure `KDB` instance with real mountpoints
-8. Add hard coded mountpoints to `KDB` instance
-
-> **Note:** Despite sharing a name, the `open` operation **does not** call the `elektra<Plugin>Open` function of any plugin (except within the bootstrap `get` operation).
+3. Run bootstrap `get` operation:
+   This loads the contents of `system:/elektra/mountpoints` so that the mountpoints can be configured.
+4. Process contract and set up plugins for hooks (see [Hooks](hooks.md))
+5. Parse mountpoints:
```

Comment:
Ideally give some links to the code (e.g. function name)

Reply:
The link is actually there, but the other way around. The `kdb*` functions have `// Step X` comments that refer to these steps.

Reply:
More important are the links from docu to code (traceability links) because usually people start looking at the docu and then would like to dive into the implementation.

Reply:
Yes, and that is fine here IMO. This is contributor documentation. You read this and then go look at `kdbGet`/`kdbSet` code and will see references to the steps you already now.

There is also no better solution. Not every step is a separate function (*) and referring to line number is obviously bad. What I will do, however, is mention that the `// Step X` comments exist.

(*) Making every step a separate function would create unnecessary friction in the code IMO. You need good names for everything, sometimes you need to pass a lot of arguments, you need to encode different results into return codes and then check those instead of just doing the cleanup directly, etc.